it/test/repositories/SessionRepositorySpec.scala

```diff
@@ -164,6 +164,18 @@ class SessionRepositorySpec
       updatedRecord.data mustEqual expectedJson
     }
 
+    "must clear data for WithdrawAdviceJourney" in {
```

Comment:
Does this test actually test that? Doesn't it only test that we don't clear data that isn't part of the journey. We have no test data from the journey involved in the test.

Reply:
sorted